stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -0,0 +1,510 @@
+                
+                <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-display-mode-3" name="rating-display-mode" value="3" disabled>
+                <label class="fd-rating-indicator__label" for="rating-display-mode-3"></label>
+                
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-display-mode-4" name="rating-display-mode" value=4" disabled>
+                <label class="fd-rating-indicator__label" for="rating-display-mode-4"></label>
+                
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-display-mode-5" name="rating-display-mode" value="5" disabled>
+                <label class="fd-rating-indicator__label" for="rating-display-mode-5"></label>
+            </div>
+            <span class="fd-rating-indicator__dynamic-text">(2 of 5)</span>
+        </div>
+    </div>
+`;
+
+DisplayMode.parameters = {
+    docs: {
+        storyDescription: `
+To make the \`Rating Indicator\` component display mode needs to be added class \`.is-display-mode\` to the \`fd-rating-indicator\` element.
+`
+    }
+};
+
+export const Focused = () => `    
+    <div class="example-container">
+        <div class="fd-rating-indicator">
+            <div class="fd-rating-indicator__container is-focus" aria-label="Star Rating (out of 5)">
+                <input aria-label="1 star" type="radio" class="fd-rating-indicator__input" id="rating-focused-1" name="rating-focused" value="1">  
+                <label class="fd-rating-indicator__label" for="rating-focused-1"></label>
+                
+                <input aria-label="2 star" type="radio" class="fd-rating-indicator__input" id="rating-focused-2" name="rating-focused" value="2" checked>
+                <label class="fd-rating-indicator__label" for="rating-focused-2"></label>
+                
+                <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-focused-3" name="rating-focused" value="3">
+                <label class="fd-rating-indicator__label" for="rating-focused-3"></label>
+                
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-focused-4" name="rating-focused" value=4">
+                <label class="fd-rating-indicator__label" for="rating-focused-4"></label>
+                
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-focused-5" name="rating-focused" value="5">
+                <label class="fd-rating-indicator__label" for="rating-focused-5"></label>
+            </div>
+            <span class="fd-rating-indicator__dynamic-text">(2 of 5)</span>
+        </div>
+    </div>
+`;
+
+Focused.parameters = {
+    docs: {
+        storyDescription: `
+By default there is built-in focus indicator for \`Rating Indicator\` component, which is not supported by IE11.
```

Comment:
"By default, rating indicator is focusable. However, it is not supported by IE11. To display a focusable rating indicator on IE11 _(browsers?)_, add the `.is-focus` class to the component."

Reply:
Not sure if it's just IE11 browsers, or applications? I'll let you be the judge of what should go there.

Reply:
I think we can just use IE11 here without prefixes browsers or applications

Reply:
Okay sounds good.